static-assets/components/cstudio-common/common-api.js

```diff
@@ -8374,33 +8374,29 @@ var nodeOpen = false,
       isReviewer: function (cb) {
         var callback = {
           success: function (data) {
-            var roles = data,
-              isRev = false,
-              topRoles = false;
-            for (var i = 0; i < roles.length; i++) {
-              if (
-                roles[i].toLocaleLowerCase() == 'admin' ||
-                roles[i].toLocaleLowerCase() == 'developer' ||
-                roles[i].toLocaleLowerCase() == 'publisher' ||
-                roles[i].toLocaleLowerCase() == 'author'
-              ) {
-                topRoles = true;
-              }
-              if (roles[i].toLocaleLowerCase() == 'reviewer' && !topRoles) {
-                isRev = true;
+            var permissions = data.permissions;
+            var hasWritePermission = false;
+            for (var i = 0; i < permissions.length; i++) {
+              if (permissions[i].toLowerCase() === 'write') {
```

Comment:
@rart we must put these permissions into constants as they change over time.

Reply:
@phuongnq let's see what @rart says about the constant and then decide what to do.

Reply:
@sumerjabri I updated to use constant defined in `common-api.js`. Kindly check this again.